data/campaigns/Heir_To_The_Throne/scenarios/05b_Isle_of_the_Damned.cfg

```diff
@@ -34,6 +34,95 @@
         [/objectives]
     [/event]
 
+    # This appoints the Advisor role to our most advanced unit coming out of The Bay of Pearls. This would be
+    # the same as the initial Advisor we would appoint if we took the path through Muff Malal's Peninsula, and
+    # will be the initial Advisor recalled at the start of The Siege of Elensefar.
+    #
+    # We create a shallow copy of this Advisor, needing only the name, type and profile image. The shallow copy
```

Comment:
Instead of creating a shallow copy, you could just override the caption and profile image in the `[message]` tag, couldn't you? Make the boat itself the speaker (to handle the scroll-to aspect).